src/utils/array.ts

```diff
@@ -70,3 +70,16 @@ export const removeItemFromArray = <T>(itemToRemove: T, array: T[]): T[] =>
 export const mergeTwoArraysUniquely = <T>(array1: T[], array2: T[]): T[] => {
   return Array.from(new Set(array1.concat(array2)));
 };
+
+/**
+ * Return a unique array of objects with no dublicates based on an object property
```

Comment:
typo: "dublicates" -> "duplicates". I suggest installing `streetsidesoftware.code-spell-checker` extension which highlights spelling mistakes.

Reply:
I have it, it was disabled though :D